internal/cmd/integrate/integrate.go

```diff
@@ -105,11 +105,29 @@ func (c *integrateCmd) run() error {
 }
 
 func setupGitIgnore(projectDir string) error {
+	// Files to ignore for all build systems
 	filesToIgnore := []string{
-		"/.cifuzz-build/",
 		"/.cifuzz-corpus/",
 		"/.cifuzz-findings/",
-		"/CMakeUserPresets.json",
+	}
+
+	buildSystem, err := config.DetermineBuildSystem(projectDir)
+	if err != nil {
+		return err
+	}
+	switch buildSystem {
+	case config.BuildSystemCMake:
+		filesToIgnore = append(filesToIgnore, []string{
+			"/.cifuzz-build/",
+			"/CMakeUserPresets.json",
+		}...)
```

Comment:
You don't have to create a slice to pass multiple arguments to `append`:
```suggestion
		filesToIgnore = append(filesToIgnore,
			"/.cifuzz-build/",
			"/CMakeUserPresets.json",
		)
```